### ISSUE TITLE
Исчезает navbar при возврате на inline-страницу

### ISSUE THREAD

**Issue by shastox** [framework7io/framework7#2651]
У нас есть единственная (стартовая) страница, которая вшита изначально в HTML. Вот ее структура:

```
<div class="view view-main ios-edges">
        <div data-name="home" class="page">
            <div class="navbar">
                <div class="navbar-inner">... </div>
            </div>
            <div class="page-content">
             ...   
            </div>
        </div>
 </div>

```

Также блок **<div data-name="home" class="page">** можно вынести и за **<div class="view view-main ios-edges">** - это не влияет на результат.

В routes: 

```
        {
            path : '/',
            el : $('.page[data-name="home"]'),
         ...
        }
```

И главное view:
```
var mainView = app.views.create('.view-main', {
    url : '/'
});
```

Если мы уходим на **page2**, потом на **page3**, а потом жмем "назад" и еще раз "назад", то возникает проблема:  в iOS теме нет **navbar'a**. Т.е. страница отображается так, как будто его нет.  Проблема свойственна именно для inline страниц. В Китчене устроено не так, сравнить не с чем.

Я добавил эвент на этот route:

```
pageMounted: function (event, page) {
                    console.log( $('.page[data-name="home"]').html());
                }
```

После два раза "назад", **navbar'a** нет в выводе в консоле для iOS, а для Android - есть. Возможно, проблема актуальна и для **toolbar'a.**

**Comment by nolimits4web** [comment]
page2 и page3 как грузятся, также как инлайн?

**Comment by shastox** [comment]
Нет, они грузятся через url. Inline только одна страница, стартовая.

**Comment by shastox** [comment]
Владимир, я набросал тестовый минимальный пример. Сделал 3 inline-страницы. Проблема только с toolbar и только в iOS-теме. 

Как воспроизвести:

1. Ставим тему для iOS в app.js.
2. Переходим на Page2.
3 Переходим нa Page3.
4. Жмем два раза "Back".

В итоге возвращаемся на первую страницу, но Toolbar скрыт.

[app.zip](https://github.com/framework7io/framework7/files/2372245/app.zip)

**Comment by shastox** [comment]
Ой, извиняюсь. Проблема с Navbar. Опечатка.

**Comment by shastox** [comment]
Спасибо большое. Но в md-теме с Navbar’om все хорошо, фреймворк его где-то сохраняет у себя, а потом, после двойного возврата на page1, снова вставляет в DOM.

**Comment by nolimits4web** [comment]
Потому что в MD теме navbar всегда внутри Page

**Comment by nolimits4web** [comment]
> Предлагаю указать, что el используется только с stacked page.

Указал

**Comment by shastox** [comment]
Извините, но все же немного добавлю демагогии.

https://github.com/framework7io/framework7-website/commit/82188ced93f99d34858ae2e65789b086219c0986

Какой тогда вообще смысл в `el`, если этот элемент должен быть по сути страницей согласно вашему комментарию: https://github.com/framework7io/framework7/issues/2651#issuecomment-420941464

Иными словами, это бесполезно, нужно просто использовать `pageName`. В ином случае, согласно документации, страница не обязана иметь pageName: https://framework7.io/docs/page.html но мне кажется это плохая практика - не использовать `pageName `и использовать `el`.
